bradd/bradd_wiki/wiki_bradd27_blend279.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
```

Comment:
Why not adding the documentation directly on the [wiki](https://github.com/pisuke/brad/wiki)?